### ISSUE TITLE
missing gear visuals

### ISSUE THREAD

**Issue by stevomatic** [HomepointXI/Issues#278]
**Date & Time**: 

**Client Version (use `/ver` in game):**

30180904_1

**Character Name:**

Kommissar

**Discord Name (if different from in game):**

**Nation:**

windy

**Job(level)/Sub Job(level):**

**NPC or Monster or item Name:**

**Zone name:** 

any

**Coordinates (use `!where`):**

**Item tool link (for items issues only):**

**Multi-boxing? (multiple clients on same connection):**

**Steps to reproduce:**
other pc's gear doesn't display, they appear in their rse. I think it started after Halkuras fixed my VW KI. I made a ticket previously but closed it because it appeared to be fixed, but later I zoned and lo and behold it was back, so it happens intermittently and so far I see no rhyme or reason why. Seemed like it works sometimes in Abyssea zones. Σ(ﾟд´;ﾉ)ﾉ
![kommissar_2018 10 16_152902](https://user-images.githubusercontent.com/26309049/47703230-f8a78a00-dbf5-11e8-9d33-d31090f33b8d.png)

**Any additional information:**

**Comment by EpicTaru** [comment]
Why'd you close the other ticket that had the same issue, @stevomatic?

**Comment by stevomatic** [comment]
> I made a ticket previously but closed it because it appeared to be fixed, but later I zoned and lo and behold it was back

Don't take this the wrong way but either you're trolling me or you comment without reading the op lol

**Comment by nfaletra** [comment]
Is style lock on?

**Comment by nfaletra** [comment]
Make sure it's not a plugin/addon. Unload all addons and see if it still happens.

**Comment by stevomatic** [comment]
![kommissar_2018 11 07_060254](https://user-images.githubusercontent.com/26309049/48127951-080c8000-e253-11e8-800e-1b7062c1cdba.png)
![sonic_2018 11 07_060309](https://user-images.githubusercontent.com/26309049/48127966-0e9af780-e253-11e8-9591-e4e20817ae27.png)
Well I don't use stylelock or gearlock plugin or mess with dats, but I tried /unloadall anyways and it didn't do anything. Also under the stock config menu there is a toggle for stylelock and that didn't do anything. Under the macro menu there's stylelock stuff I messed with and it did nothing either. Also loaded the gearlock plugin to see if it would make anything happen but no luck. I have no idea how such things work so I'm pretty much out of ideas except try and see if a fresh install somehow helps
